src/main/java/org/opentripplanner/framework/time/TimeUtils.java

```diff
@@ -246,4 +248,8 @@ public static long busyWait(int waitMs) {
     }
     return value;
   }
+
+  public static int toTransitTimeSeconds(ZonedDateTime transitSearchTimeZero, Instant time) {
```

Comment:
```suggestion
  /**
   * Calculate the relative time in seconds with the given {@code transitSearchTimeZero} as the
   * base. There is no restriction on the returned time, it can be in the past(negative) and
   * many days ahead of the base. This method can be used to translate an API instance of time
   * into the OTP internal transit model time, when the search zero-point-in-time is known.
   */
  public static int toTransitTimeSeconds(ZonedDateTime transitSearchTimeZero, Instant time) {
```